pubspec.yaml

```diff
@@ -31,28 +31,34 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  flutter_localizations:
+    sdk: flutter
 
 
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  built_value: ^8.4.1
```

Comment:
I suggest using strict packages version, without `^` sign. It will be less error-prone